src/main/java/pl/byczazagroda/trackexpensesappbackend/service/WalletService.java

```diff
@@ -7,4 +7,6 @@
 
 public interface WalletService {
     WalletDTO createWallet(@Valid CreateWalletDTO createWalletDTO);
+
+    WalletDTO findById(@Valid long id);
```

Comment:
Do we use Bean validation for `long id'` field? If not then it's better to remove this redundant `@Valid` annotation